db/explorerParser.ts

```diff
@@ -0,0 +1,143 @@
+// This code was adapted from https://github.com/owid/automation/blob/main/automation/mirror_explorers.py
+
+interface Statement {
+    verb: string
+    args: string[]
+    block?: Record<string, any>[]
+}
+
+const JSON_VERSION = 1
+
+// Added FLAGS constant from mirror_explorers.py
```

Comment:
Didn't have time for a full review, but just remarking this here:
In the owid-grapher codebase, you can take these directly from https://github.com/owid/owid-grapher/blob/f51c2a9092d9c94a68e422c05836f8fc753c1230/packages/@ourworldindata/explorer/src/ExplorerGrammar.ts - just need to strip out the special ones that contain `headerCellDef`, I think.